gisserver/operations/base.py

```diff
@@ -258,14 +258,16 @@ def get_parameters(self):
 
     def _parse_output_format(self, value) -> OutputFormat:
         """Select the proper OutputFormat object based on the input value"""
-        value = value.replace(" ", "+")  # allow application/gml+xml on the KVP.
-        try:
-            return next(o for o in self.output_formats if o.matches(value))
-        except StopIteration:
-            raise InvalidParameterValue(
-                "outputformat",
-                f"'{value}' is not a permitted output format for this operation.",
-            ) from None
+        # The str.replace is here to "allow application/gml+xml on the KVP",
+        # but I'm not sure what that comment was supposed to mean.
```

Comment:
Hi @lbam :-) that comment was a bit vague indeed. The problem is that adding "application/gml+xml" in the request GET queryparameters, that will be parsed as a space-character instead. The "KVP" stands for the key-value-pair input, which is WFS speak for the querystring parameters.

Reply:
I'd figured it had something to do with the parsing of query params. I'll leave this in for now. Thanks for the heads up!